src/components/IconButtonBar/IconButtonBar.js

```diff
@@ -88,12 +92,7 @@ const IconButtonBar = ({
             direction={overflowMenuDirection}
             flipped
             menuOptionsClass={iconButtonBarMenuOptionsClasses}
-            renderIcon={() =>
-              renderIconButton({
-                renderIcon: getOverflowMenuIcon(),
-                tooltip: false,
-              })
-            }
+            renderIcon={getOverflowMenuIcon()}
```

Comment:
I noticed that this component was rendering our `IconButton`, versus rendering the button that Carbon actually uses in the `OverflowMenu` component, which was causing some issues with our own `IconButton` classes & styles overriding their `OverflowMenu`-related classes. So that's why I made this change, which also means we have to do fewer style overrides 👍